### ISSUE TITLE
Assets are missing from dist

### ISSUE THREAD

**Issue by mateus** [photonstorm/phaser3-project-template#32]
I am trying to deploy my code to Firebase Hosting and the assets are not present in the dist folder. Any suggestions on how to fix that from the `yarn build` command? 

@photonstorm

**Comment by zvermafia** [comment]
Same problem with `npm run build`...

**Comment by mateus** [comment]
I used `copy-webpack-plugin` to set the assets folder correctly in the `prod.js` webpack config to make it work.

```
module.exports = merge(base, {
  ...
  plugins: [
    new CopyPlugin([
      { from: 'assets', to: 'assets' }
    ])
  ],
}
```

**Comment by chrisl8** [comment]
Are you adding the files inside of your code by importing them?
For instance, to use the `fullscreen.png` file you would import it like so in your code:

`import fullscreen from '../assets/spriteSheets/fullscreen.png';`

Then later in your code you can use the variable `fullscreen` instead of the filename.

Then when you perform a build, webpack will bundle this file, and all of the other imported files, for you.

**Comment by jasonbarry** [comment]
I made a PR that fixes asset resolution on both dev and prod webpack configurations.

**Comment by thebharatbhushan** [comment]
can i make a build without import. directly use this.load.image("bg","img/bg.jpg");

**Comment by photonstorm** [comment]
This is fixed with v2.0.0 of this template.
